utils/ansible/ftest/templates/daos-make.sh.j2

```diff
@@ -197,7 +197,7 @@ if [[ ! -d "$DAOS_INSTALL_DIR" ]] ; then
 fi
 
 cd "$DAOS_SOURCE_DIR"
-SCONS_OPTS="-j $JOBS_NB"
+SCONS_OPTS="-j $JOBS_NB --implicit-deps-changed"
```

Comment:
Not sure this was meant to be included in the commit, seems unrelated?

Reply:
Yes and No, I needed to properly run ftest on my little cluster. Not sure, it deserves a dedicated PR.